pinc/charsuite-polytonic-greek.inc

```diff
@@ -93,58 +91,68 @@ $charsuite->reference_urls = [
 ];
 
 $pickerset = new PickerSet();
+
 $pickerset->add_subset(utf8_chr('U+0391') . '-' . utf8_chr('U+03a9'), [
     [ 'U+0391-U+03a1', NULL, 'U+03a3-U+03a9' ], # capital alpha through omega
     [ 'U+03b1-U+03c9' ], # lowercase alpha through omega
 ]);
-$pickerset->add_subset(utf8_chr('U+0386'), [
-    [ 'U+0386', 'U+1fba', NULL, 'U+1f08', 'U+1f09', 'U+1f0c', 'U+1f0d', 'U+1f0a', 
-      'U+1f0b', 'U+1f0e', 'U+1f0f', 'U+1fbc', NULL, NULL, NULL, 'U+1f88', 'U+1f89',
-      'U+1f8c', 'U+1f8d', 'U+1f8a', 'U+1f8b', 'U+1f8e', 'U+1f8f'  ],
+
+$pickerset->add_subset(utf8_chr('U+03ac'), [
+    [ NULL, NULL, NULL, 'U+1f08', 'U+1f09', 'U+1f0c', 'U+1f0d', 'U+1f0a', 
+      'U+1f0b', 'U+1f0e', 'U+1f0f', NULL, NULL, NULL, NULL, NULL, NULL,
```

Comment:
I would prefer we not try to fill in the right side of the picker sets with `NULL`s. If we want the picker set rows "filled out" on the `charsuite.php` page we should do it there.